### PR TITLE
Modify S2612: add Ansible

### PR DESCRIPTION
**Pull request by sebastien-andrivet-sonarsource**
## Review

A dedicated reviewer checked the rule description successfully for:

- [ ] logical errors and incorrect information
- [ ] information gaps and missing content
- [ ] text style and tone
- [ ] PR summary and labels follow [the guidelines](https://github.com/SonarSource/rspec/#to-modify-an-existing-rule)

### CODE REVIEW

File: rules/S2612/ansible/metadata.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  "tags": [
+    "cwe",
+    "docker"
+  ],
+  "securityStandards": {
+    "CERT": [
+
+    ],
+    "CWE": [
+      732
```
Comment (pierre-loup-tristant-sonarsource) [inline rules/S2612/ansible/metadata.json:11]: What's the rationale for removing CWE-266: Incorrect Privilege Assignment?

File: rules/S2612/ansible/rule.adoc [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    - name: Create /etc/demo with permissions
+      ansible.builtin.file:
+        path: /etc/demo
+        state: directory
+        mode: '0777'
+
+    - name: Copy demo3.conf without setting permissions (default permissions)
+      ansible.builtin.copy:
+        src: /files/demo.conf
+        dest: /etc/demo/demo.conf
```
Comment (pierre-loup-tristant-sonarsource) [inline rules/S2612/ansible/rule.adoc:32]: I made a comment in https://sonarsource.atlassian.net/browse/APPSEC-2157 to discuss if we should raise in the absence or the `mode` argument.

**Review by pierre-loup-tristant-sonarsource — CHANGES_REQUESTED** [review]
Please have a look at the discussion in https://sonarsource.atlassian.net/browse/APPSEC-2157?focusedCommentId=670196 and, if needed, remove the code example where the  `mode` argument is omitted.

File: rules/S2612/ansible/rule.adoc (hunk 2 above)
Comment (sebastien-andrivet-sonarsource) [reply rules/S2612/ansible/rule.adoc:32]: We should not.

**Review by pierre-loup-tristant-sonarsource — APPROVED** [review]
LGTM

**Comment by pierre-loup-tristant-sonarsource** [comment]
Validation of links are failing but non are in the "see" section of S2612

**Pull request merged**